packages/core/src/handlers/http.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/core
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Application} from '../application';
+import {ServerRequest, ServerResponse} from 'http';
+import {Handler} from '../internal-types';
+
+export function httpHandler(app: Application): Handler {
```

Comment:
Perhaps this should be expecting any object (interface) providing `handleHttp` method? Then we don't have to mock/stub a whole application in the tests.

```ts
interface CanHandleHttp {
  handleHttp(req: ServerRequest, res: ServerResponse): Promise<void>;
}

export function httpHandler(app: CanHandleHttp): Handler {
  // ...
}
```